src/examples/accordions/timeline/TimelineAlternating.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Timeline } from '@zendeskgarden/react-accordions';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Span } from '@zendeskgarden/react-typography';
+
+const Example = () => (
+  <Row justifyContent="center">
+    <Col sm={9}>
+      <Timeline isAlternate>
+        <Timeline.Item>
+          <Timeline.Content>
+            <Span isBold style={{ display: 'block' }}>
```

Comment:
For repeat components like this, we tend to break out a Styled component. See https://garden.zendesk.com/components/notifications#placement for comparison. Maybe something like:

```jsx
const StyledSpan = styled(Span).attrs({ isBold: true })`
  display: block;
`;
```